docs/cmdlet-example/command-line-simple-example.md

```diff
@@ -0,0 +1,224 @@
+# Creating a cross-platform binary module with the .NET Core command-line interface tools
+
+This example uses the [.NET Core command-line interface tools][dotnet-cli] (aka
+`dotnet` CLI) to demonstrate how to create a binary module that is portable across operating
+systems supported by **PowerShell Core** as well as **Windows PowerShell** version 3 and higher.
+
+Because the binary module's assembly will be created as a .NET Standard 2.0 class library,
+the same assembly can be imported into both PowerShell Core and Windows PowerShell.
+This means you do not have to build and distribute separate assemblies that target these two
+different implementations of PowerShell.
+
+## Prerequisites
+* PowerShell Core and/or Windows PowerShell
+
+  For this example, you can use any operating system that is supported by PowerShell Core.
+  To see if your operating system is supported and to get instructions on how to install
+  PowerShell Core on your operating system, see the [Get PowerShell][pscore-os] topic in
+  the PowerShell repo's [README.md][readme] file.
+
+  Note: On Windows 10 Anniversary Update or higher, you can use the [Windows Subsystem for
+  Linux][wsl] (WSL) console to build the module. In order to import and use the module, you'll need
+  to install PowerShell Core for the distribution and version of Linux you're running.
+  You can get that version info by running the command `lsb_release -a` from the WSL console.
+
+* .NET Core 2.0 SDK
+
+  Download and install the [.NET Core 2.0 SDK][net-core-sdk] for your operating system.
+  It is recommended that you use a package manager to install the SDK on Linux.
+  See these [instructions][linux-install] on how to install the SDK on Linux.
+  Be sure to pick your distribution of Linux e.g. RHEL, Debian, etc to get the
+  appropriate instructions for your platform.
+
+## Create the .NET Standard 2.0 Binary Module
+
+1. Verify you are running the 2.0.0 version of the `dotnet` CLI.
```

Comment:
make sure you have added this file to the markdown syntax tests.

Reply:
Done